store/tikv/lock_resolver.go

```diff
@@ -551,7 +576,7 @@ func (e txnNotFoundErr) Error() string {
 // getTxnStatus sends the CheckTxnStatus request to the TiKV server.
 // When rollbackIfNotExist is false, the caller should be careful with the txnNotFoundErr error.
 func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte,
-	callerStartTS, currentTS uint64, rollbackIfNotExist bool, forceSyncCommit bool) (TxnStatus, error) {
+	callerStartTS, currentTS uint64, rollbackIfNotExist bool, forceSyncCommit bool, lockInfo *Lock) (TxnStatus, error) {
```

Comment:
Seems passing this `lockInfo` repeats `txnID` and `primary`.

Reply:
The interface [GetTxnStatus](https://github.com/pingcap/tidb/pull/21844/files?file-filters%5B%5D=.go#diff-5256dbab90938778e9c3792f2792bf8accf2485e8ab41b948938b2cc0c9e11c4R488) used by tidb-binlog will pass in `primary key` and the `startTS`, we could do some refactor with binlog later.